spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -116,6 +116,43 @@
       end
     end
 
+    context 'locked out session' do
+      let(:locked_at) { Time.zone.now }
+      let(:user) { create(:user, :fully_registered) }
+      let(:locked_out_time_remaining) do
+        distance_of_time_in_words(Time.zone.now, (locked_at + 600.seconds), true)
```

Comment:
1. Unless we freeze time, I wouldn't trust that this wouldn't be flakey
2. Should this be using the `IdentityConfig.store.max_bad_passwords_window_in_seconds` config, rather than hard-coding 600 seconds?

Reply:
Yea but I think freezing time breaks distance_time_from_words too.